src/views/AddItem.css

```diff
@@ -0,0 +1,119 @@
+.timeframe {
+	display: flex;
+	flex-direction: column;
+}
+
+.timeframe > fieldset {
+	display: flex;
+	flex-direction: column;
+	padding: 20px;
+	margin: 50;
+	width: auto;
+	position: fixed;
+}
+
+.timeframe label {
+	padding: 10px;
+}
+
+.time-frame-container {
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	justify-content: center;
+	height: 40%;
+	width: full;
+	max-height: 40vh;
+	margin: 10%;
+}
+
+.item-form-container {
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	margin-top: 2rem;
+}
+
+.item-name-container {
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+}
+
+.item-name-container > label {
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	padding: 20px;
+	font-size: 1.3em;
+	font-weight: bold;
+}
+
+.item-name-container input {
+	padding: 10px;
+	margin: 20px;
+	box-sizing: border-box;
+	border: 1px solid var(--yellow);
+}
+
+.legend {
+	font-size: 1.2em;
+	font-weight: bold;
+	min-width: 30vw;
+	max-width: full;
+}
+
+.button {
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	height: 51px;
+	border-radius: 10px;
+	margin: 10px;
+}
+
+.button > button {
+	width: 150px;
+	height: 51px;
+	background: var(--yellow);
+	border-radius: 10px;
+}
+
+p,
```

Comment:
Everything is looking great so far! I think we might just need to change this <p> tag to a different font color. It looks like the text confirmation or error when adding an item is the same color as the background, so I can't see the message that my item was added or is already on the list. Same thing is happening with "Your list name is" on the list page.